client/me/purchases/manage-purchase/index.jsx

```diff
@@ -81,8 +82,10 @@ const ManagePurchase = React.createClass( {
 		recordPageView( 'manage', this.props, nextProps );
 	},
 
-	isDataFetchingAfterRenewal() {
-		return this.props.selectedPurchase.isFetching;
+	isDataFetchingAfterChange() {
+		const changeDestinations = [ 'thank-you', 'credit-card-updated' ];
```

Comment:
`changeDestinations` is a bit of a strange name for this variable. What about `dataChangeDestinations`?